packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -116,6 +128,14 @@ const onInput = (e: Event) => {
 const clearInput = (e: Event) => {
   emit('update:specFilterModel', '')
 }
+
+const handleFocus = () => {
+  runnerUiStore.setPreference('isSpecsListInputFocused', true)
```

Comment:
Hmm I think this is actually the wrong place for this - looking at the other preferences, they are all things that are saved in the appData - like reporterWidth, reporterHeight - things that persist between sessions. 

It looks like `setPreferences` is coded in a way to just update any key in the store but I think this is actually implemented for the app preferences specifically, it's only used here right now: https://github.com/cypress-io/cypress/blob/develop/packages/app/src/composables/usePreferences.ts#L21 which is mixed in with all the GraphQL etc.

I think we might want to create a separate setter for this, like `runnerUiStore.setShouldSpecsListInputFocused` to better separate the concerns. 

Also looking *more* closely it looks like `runnerUiStore` should likely be two separate stores - `runnerUiStore` (for things like this) and `runnerPreferences` (for appData and things that are persisted).

Reply:
It did feel off using the `setPreferences`, it would make more sense if the mutation for saving a preference was in the `setPreference` action rather than having to do:
```
runnerUiStore.setPreference(preference, value)
```

Reply:
Sounds good, separate action works for me.